httptransport/server.go

```diff
@@ -25,6 +26,7 @@ const (
 	IndexAPIPath            = apiRoot + "index_report"
 	IndexReportAPIPath      = apiRoot + "index_report/"
 	IndexStateAPIPath       = apiRoot + "index_state"
+	NotificationAPIPath     = apiRoot + "notification/"
```

Comment:
OpenAPI spec defines it as plural ``/api/v1/notifications/

Reply:
fixed